microprofile/tests/testng/src/main/resources/META-INF/services/org.testng.ITestNGListener

```diff
@@ -0,0 +1 @@
+io.helidon.microprofile.tests.testng.HelidonTestNGListener
```

Comment:
This file should not be in the PR, it is generated automatically by our plugin.

Reply:
Fixed!

Reply:
If I remove it, tests stop working correctly.

Reply:
Fixed with the correct property.